docs/user-guide/release-notes/2.12.x.rst

```diff
@@ -8,11 +8,20 @@ Pulp 2.12.0
 New Features
 ------------
 
- * Task profiling can now be enabled. This will use cProfiling on an individual task and write the profile to a directory for a given task. While this can impact performance, this can enable users to get some insight into what a task is doing or use the output to give to a developer for debugging.
- * ``pulp-manage-db`` will not continue if pulp_celerybeat, pulp_resource_manager, or pulp_workers
-   processes are running. This will prevent the user from corrupting their Pulp installation by
-   migrating with running workers. This works in both standalone and clustered installations. Pulp
-   may wait up to 92 seconds to determine if workers are running.
+* Task profiling can now be enabled. This will use cProfiling on an individual task and write the
```

Comment:
Can s/cProfiling/cProfile/   and also can cProfile link to this page?  https://docs.python.org/2/library/profile.html#module-cProfile